src/codecs/encoding/format/json.rs

```diff
@@ -18,6 +20,13 @@ impl JsonSerializerConfig {
     pub const fn build(&self) -> JsonSerializer {
         JsonSerializer
     }
+
+    /// The schema required by the serializer.
+    pub fn schema_requirement(&self) -> schema::Requirement {
+        // Technically we can serialize any type of `Value` to JSON, even "non-JSON" types such as
+        // `timestamp`, but it's not a lossless serialization. Should we allow it in the schema?
```

Comment:
I agree with that. I don't think that we want to require users to go through another transformation step to convert non-native JSON `Value` types, or that any of this would be unexpected behavior.

So this needs change before merging?

Reply:
Resolved in 2af897b.